openapi.yaml

```diff
@@ -2921,6 +2940,29 @@ components:
         description:
           type: string
           example: 'My test account'
+    UpdateAwsAccount:
+      required:
+        - name
+        - accountId
+        - roleArn
+        - externalId
+      type: object
+      properties:
+        name:
```

Comment:
Properties not required need to be included in the schema as well.

Reply:
👍🏻